frontend/src/Series/MonitoringOptions/MonitoringOptionsModalContent.js

```diff
@@ -1,16 +1,20 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
+import styles from 'AddSeries/AddNewSeries/AddNewSeriesModalContent.css';
```

Comment:
Don't import CSS for another component, you'll need to create a new CSS file for this component.

Reply:
done! thanks for the guidance